src/engine/openspaceengine.cpp

```diff
@@ -1157,9 +1174,30 @@ void OpenSpaceEngine::preSynchronization() {
 
         func();
     }
+
+    if (!_hasInitializedProfile) {
+        setFromProfile_camera(*global::profile);
+        setFromProfile_additionalScripts(*global::profile);
+        _hasInitializedProfile = true;
+    }
     LTRACE("OpenSpaceEngine::preSynchronization(end)");
 }
 
+void OpenSpaceEngine::loadInitAssetSection(std::string& initAssetOutput,
```

Comment:
This function is (correctly) not called anywhere anymore and can die

Reply:
Gone